Update RecurringTask.java

mango-ops opened the pull request:
Removed freq to length value, choice of frequency is singularly(1) or weekly(7).

mango-ops committed:
Update RecurringTask.java

Removed freq to length value, choice of frequency is singularly(1) or weekly(7).

Andr3wD reviewed (changes requested):
The frequency can be 1-7 (or maybe even more), and doesn't have to be just 1 or 7 if I remember correctly. The frequency is how many days until the task repeats.

Also, the setFrequency() and other basic setters are for validating the input from code, instead of from UserHandler. This is primarily for when tasks are being loaded from JSON, they can be created with the constructor, and the constructor will call the basic setter to validate the input.

Andr3wD commented:
I was wrong about the frequency, it seems to just be 1 or 7. 
Though, the setFrequency() setter should probably be changed before merging though.

mango-ops committed:
Updated setters

raw input validation for code setters
userhandler setters don't use code setters anymore
